packages/expo/src/ErrorRecovery/ExponentErrorRecovery.web.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+  get name(): string {
+    return 'ExponentErrorRecovery';
+  },
+};
```

Comment:
I don't think we want to throw an error if someone calls this method on web since there is no notion of automatically reloading the web page. We could later expose an API to auto-reload on error, but would need to build in special support for that with a top-level React error boundary. For now, `setRecoveryProps(): void {}` is fine I think.